### PR TITLE
VEN-740 | Add Nostojärjestys card to front page

### PR DESCRIPTION
**Pull request by TatuArvela**
## Description

* Add Nostojärjestys card to front page, with adjustments to appearance
* Fix Thank you page in Finnish and English

## Related issues

* [VEN-740](https://helsinkisolutionoffice.atlassian.net/browse/VEN-740): Add Nostojärjestys card to the home page
* [VEN-789](https://helsinkisolutionoffice.atlassian.net/browse/VEN-789): Fix the application Thank you page title text (FI and ENG)

### CODE REVIEW

File: src/components/common/card/card.scss [hunk 1]
```diff
@@ -24,9 +24,25 @@
...
+
+  &__button-label {
+    font-weight: inherit;
+    white-space: pre-wrap;
+  }
+
+  &__arrow-icon {
+    margin-left: 1rem;
+    height: 1.5em;
+    vertical-align: middle;
```
Comment (santtul) [inline src/components/common/card/card.scss:44]: Style has no effect

File: src/components/pages/frontPage/frontPage.scss [hunk 2]
```diff
@@ -7,6 +8,24 @@
   }
 
   &__card-wrapper {
+    padding-right: 10px;
+    padding-left: 10px;
     margin-bottom: 2em;
+    position: relative;
```
Comment (santtul) [inline src/components/pages/frontPage/frontPage.scss:14]: Style has no effect

File: src/components/pages/frontPage/frontPage.scss [hunk 3]
```diff
@@ -7,6 +8,24 @@
...
+    width: 100%;
```
Comment (santtul) [inline src/components/pages/frontPage/frontPage.scss:15]: Style has no effect

**Review by santtul — CHANGES_REQUESTED** [review]
Looks great, some nitpicks

File: src/components/pages/frontPage/frontPage.scss (hunk 3 above)
Comment (TatuArvela) [reply src/components/pages/frontPage/frontPage.scss:15]: Not on desktop, but it has on smaller screens

File: src/components/pages/frontPage/frontPage.scss (hunk 2 above)
Comment (TatuArvela) [reply src/components/pages/frontPage/frontPage.scss]: Removed

File: src/components/common/card/card.scss (hunk 1 above)
Comment (TatuArvela) [reply src/components/common/card/card.scss]: Removed

**Review by santtul — APPROVED** [review]
👍

**Pull request merged**